src/Acuminator/Acuminator.Analyzers/FixProviders/DAC/ConstructorInDacCodeFix.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Composition;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.Editing;
+using Acuminator.Utilities;
+
+namespace Acuminator.Analyzers.FixProviders
+{
+    [Shared]
+    [ExportCodeFixProvider(LanguageNames.CSharp)]
+    public class ConstructorInDacCodeFix : CodeFixProvider
+    {
+        public override ImmutableArray<string> FixableDiagnosticIds { get; } =
+            ImmutableArray.Create(Descriptors.PX1028_ConstructorInDacDeclaration.Id);
+        public override FixAllProvider GetFixAllProvider() => WellKnownFixAllProviders.BatchFixer;
+
+        public override Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            return Task.Run(() =>
+            {
+                var diagnostic = context.Diagnostics.FirstOrDefault(d => d.Id == Descriptors.PX1028_ConstructorInDacDeclaration.Id);
+
+                if (diagnostic == null || !diagnostic.IsRegisteredForCodeFix())
```

Comment:
@saloponov, IsRegisteredFoCodeFix is redundant here since you don't pass any properties to the DIagnostic.Create method in analyzer